sqlmesh/migrations/v0051_rename_column_descriptions.py

```diff
@@ -0,0 +1,61 @@
+"""Rename the node attribute `column_descriptions_` to `column_descriptions` in snapshots."""
+
+import json
+
+import pandas as pd
+from sqlglot import exp
+
+from sqlmesh.utils.migration import index_text_type
+
+
+def migrate(state_sync, **kwargs):  # type: ignore
+    engine_adapter = state_sync.engine_adapter
+    schema = state_sync.schema
+    snapshots_table = "_snapshots"
+    if schema:
+        snapshots_table = f"{schema}.{snapshots_table}"
+
+    new_snapshots = []
+    found_col_descriptions = False
+
+    for name, identifier, version, snapshot, kind_name, expiration_ts in engine_adapter.fetchall(
+        exp.select("name", "identifier", "version", "snapshot", "kind_name", "expiration_ts").from_(
+            snapshots_table
+        ),
+        quote_identifiers=True,
+    ):
+        parsed_snapshot = json.loads(snapshot)
+
+        column_descriptions = parsed_snapshot["node"].pop("column_descriptions_", None)
+        if column_descriptions:
```

Comment:
Should this be `is None`? I'm concerned that if it's an empty dict for whatever reason, we won't migrate.

Reply:
Actually, checking the existence of the key will be the safest option.

Reply:
Ah, good call